app/layout.tsx

```diff
@@ -1,18 +1,23 @@
-import './globals.css'
+import "./globals.css";
 
 export const metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-}
+  title: "World Holiday",
+  description: "Generated by create next app",
+};
 
 export default function RootLayout({
   children,
 }: {
-  children: React.ReactNode
+  children: React.ReactNode;
 }) {
   return (
     <html lang="en">
-      <body>{children}</body>
+      <body>
+        {children}
+        <footer>
+          <span>Developer by Ting Chieh Lin</span>
```

Comment:
grammar: here the correct english is "Developer by *",

suggestion: You could add the URL to your github :)